app/views/content_items/_sticky_nav.html.erb

```diff
@@ -5,3 +5,14 @@
     </div>
   </div>
 </div>
+
+<script>
```

Comment:
We might be better off using https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#stick-at-top-when-scrolling for the sticky nav unless there's a particular reason why not?

Reply:
That's a good shout @sihugh, we decided to write our own implementation for educational purposes, but we can see the benefit of use that instead and we will change it afterwards (another PR).